src/pages/Settings/sections/VoiceCloneSection.css

```
@@ -0,0 +1,544 @@
+  transform: translate(-50%, -50%);
+  width: 90%;
+  max-width: 520px;
+  max-height: 90vh;
+  overflow-y: auto;
+  background: var(--card-bg);
+  border-radius: 16px;
+  z-index: 1000;
+  animation: slideUp 0.3s ease;
+}
+
+@keyframes fadeIn {
```

Stylelint (17.5.0)

[error] Expected keyframe name "fadeIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)